### DISCUSSION TITLE
How can i fix this ? . pls help me . plsssssssssssss

### DISCUSSION THREAD

**Discussion by luongcongtandpt** [Kas-tle/java2bedrock.sh#10787] [Converter Help]
I can't convert rsp . here is download link : https://www.dropbox.com/scl/fi/l2l92fpeq90vfos2ttpk6/generated.zip?rlkey=1g5bewo7b11tw7n5ii4oxqh5w&dl=0
![image](https://github.com/Kas-tle/java2bedrock.sh/assets/137200888/611a60f1-7259-4f47-ae8a-c29900169d1d)

**Comment by typicalsmc** [comment]
try using discord instead of dropbox, it will need a direct download link
